contacts.py

```diff
@@ -62,31 +70,40 @@ def _get_parts_from_filename(file_name):
             serial_number = int(parts[2])
         return code, floating_seconds, serial_number
 
+    def _add_to_items(self, dirs, key, floating_seconds_and_serial_number):
+        contact_dates = self.items[dirs[0]][dirs[1]][dirs[2]]  # { key: [(floating_seconds, serial)]
+        if key in contact_dates:  # Already at least one item for this key
+            contact_dates[key].append(floating_seconds_and_serial_number)
+        else:
+            contact_dates[key] = [floating_seconds_and_serial_number]
```

Comment:
needs to store this back into self.items in this branch since prior to this store contact_date[keys] will be a ContactDict

Reply:
Why? Why would what is in contact_dates[key] sometimes be a ContactDict and sometimes a list ?

Reply:
contact_dates is `self.items[dirs[0]][dirs[1]][dirs[2]]` which if items is `{ AA: { BB: { CC: AABBCCDEF123: [(floatingseconds, serialnumber)] } } }` is always going to be a list (or there will be no entry for key)